sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=CMCLFE
+sonar.projectName=CMC :: Legal Frontend
+sonar.projectVersion=0.0.1
```

Comment:
Did ask to remove this as well as this makes maintenance just harder. On Jenkins to could be probably overridden.

Reply:
removed version as well